http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -69,8 +69,8 @@ object KubernetesServiceDbQueries {
   def getActiveFullAppByName(googleProject: GoogleProject, appName: AppName, labelFilter: LabelMap = Map())(
```

Comment:
Changing this query's behavior slightly: it now returns non-deleted apps even if the underlying cluster/nodepool is deleted. This can happen since we clean up resources when an app enters `Error` status -- we still want the app to be retrievable in this case.

Reply:
That makes sense to me. The method is used in several places. Haven't checked if there may be an adverse effect in any of the call site. Hopefully the tests would catch it if there is ™️ .

Reply:
Yeah I'm a bit nervous since this is used in so many places, but I think the new behavior is correct, and I'll check all tests + test manually in the UI :)